src/ldc/eh/common.d

```diff
@@ -429,7 +454,7 @@ extern(C) auto eh_personality_common(NativeContext)(ref NativeContext nativeCont
     // The instruction pointer (ip) will point to the next instruction after
     // whatever made execution leave this frame, so substract 1 for the range
     // comparison below.
-    immutable ptrdiff_t ip = nativeContext.getIP() - 1;
+    ptrdiff_t ip = nativeContext.getIP() - 1;
```

Comment:
I just realized that `ip` is not used in any common code between SjLj and dwarf versions.  I think I will copy this line into each version block and leave it immutable for dwarf version.